src/renderers/art/ReactARTFiber.js

```diff
@@ -408,7 +408,7 @@ const ARTRenderer = ReactFiberReconciler({
     // Noop
   },
 
-  commitUpdate(instance, oldProps, newProps) {
+  commitUpdate(instance, type, oldProps, newProps) {
```

Comment:
Nice! This also eliminates the need for `ReactNativeFiber` renderer to bundle along the `viewConfig` with each instance as well.